chipy_org/apps/meetings/models.py

```diff
@@ -249,8 +300,13 @@ class RSVP(CommonModel):
     last_name = models.CharField(max_length=MAX_LENGTH, blank=True, null=True)
     first_name = models.CharField(max_length=MAX_LENGTH, blank=True, null=True)
     email = models.EmailField(max_length=255, blank=True, null=True)
+
     meeting = models.ForeignKey(Meeting, on_delete=models.CASCADE)
-    response = models.CharField(max_length=1, choices=RSVP_CHOICES)
+
+    response = models.CharField(max_length=1, choices=RSVP_RESPONSE, default=YES)
+    status = models.CharField(max_length=1, choices=RSVP_STATUS, blank=True, null=True)
+    venue = models.CharField(max_length=1, choices=RSVP_VENUE, default=IN_PERSON)
```

Comment:
Would this be clearer with `attendance_type` or `attendance_method`? We already have a domain concept for `venue` that this could be easily confused with. 

(Case study: I did confuse it while reading this PR)

Reply:
Also, this feels like it could be merged with response. The types of response seem to be `attending_in_person, attending_virtually, not_attending` rather than decoupling these two data points.

Reply:
I didn't like "venue"  when I wrote it.  I like attendance_type much better.

Reply:
merged to response only.